.github/workflows/ci.yml

```
@@ -3,166 +1,218 @@
-        test -d src/madspark/agents || (echo "Missing src/madspark/agents directory" && exit 1)
-        test -f src/madspark/agents/__init__.py || (echo "Missing src/madspark/agents/__init__.py" && exit 1)
-        echo "✅ Project structure validation passed"
+      - uses: actions/checkout@v4
+      
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      
+      - name: Validate Docker Compose
+        run: |
+          docker compose -f web/docker-compose.yml config
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)